### ISSUE TITLE
Do you have the LAMBADA performance for gpt-bert-strict-small?

### ISSUE THREAD

**Issue by sinking-point** [ltgoslo/gpt-bert#5]
I'd be very interested in this for a video I'm making, but it doesn't seem to be included in the paper from what I can see.

**Comment by davda54** [comment]
Hi, thanks for your interest! :) As LAMBADA was not part of the BabyLM evaluation suite, we did not evaluate it on the small model. We only used it for the ablation study that compared different causal-masked ratios.

**Comment by sinking-point** [comment]
Okay no worries, thanks for the quick response.  I've been playing around with the model and it has really impressive grammar for such a limited training set.
